docs/index.md

```diff
@@ -1,3 +1,40 @@
 # paperbee
```

Comment:
This file is not needed, please delete it. README.md in the root should be updated according to the other messengers' description